literate_notebooks/src-ES/13_extras.jl

```diff
@@ -1,175 +1,179 @@
 # # Introduction to DataFrames
+# # Introducción a DataFrames
 # **[Bogumił Kamiński](http://bogumilkaminski.pl/about/), May 13, 2018**
+# (Traducción de Miguel Raz Guzmán Macedo, 18 de Abril de 2021)
 
 using DataFrames
 
-# ## Extras - selected functionalities of selected packages
+# ## Extras - funcionalidades especiales de paqueterías selectas
 
 #-
 
-# ### FreqTables: creating cross tabulations
+# ### FreqTables: crear tabulaciones cruzadas
 
 using FreqTables
 df = DataFrame(a=rand('a':'d', 1000), b=rand(["x", "y", "z"], 1000))
-ft = freqtable(df, :a, :b) # observe that dimensions are sorted if possible
+ft = freqtable(df, :a, :b) # nota: las dimensiones se ordenan si es posible
 
 #-
 
-ft[1,1], ft['b', "z"] # you can index the result using numbers or names
+ft[1,1], ft['b', "z"] # puedes indexar el resultado con números o nombres
 
 #-
 
-prop(ft, 1) # getting proportions - 1 means we want to calculate them in rows (first dimension)
+prop(ft, 1) # obtener tamaños - 1 significa que lo queremos calcular en filas (primera dimensión)
+
 
 #-
 
-prop(ft, 2) # and columns are normalized to 1.0 now
+prop(ft, 2) # y las columnas se normalizan a 1.0
 
 #-
 
 x = categorical(rand(1:3, 10))
-levels!(x, [3, 1, 2, 4]) # reordering levels and adding an extra level
-freqtable(x) # order is preserved and not-used level is shown
+levels!(x, [3, 1, 2, 4]) # reorganizar niveles y añadir un nivel extra
+freqtable(x) # el orden se manteine y los nivelos no usados se meustran
 
 #-
 
-freqtable([1,1,2,3,missing]) # by default missings are listed
+freqtable([1,1,2,3,missing]) # por default los valores faltatnes (`missing`s) se muestran en lista
```

Comment:
faltantes, instead of faltatnes

Reply:
yup, y "se muestran como lista" me suena mejor.